setup.py

```diff
@@ -67,7 +67,7 @@ def _get_dbt_core_version():
     include_package_data=True,
     install_requires=[
         "dbt-core~={}".format(dbt_core_version),
-        "snowflake-connector-python[secure-local-storage]==2.8.3",
+        "snowflake-connector-python[secure-local-storage]==2.8.2",
```

Comment:
Something to note is that this version was yanked on PyPI:
https://pypi.org/project/snowflake-connector-python/#history
<img width="162" alt="image" src="https://user-images.githubusercontent.com/44704949/219757484-4c5fda0b-9537-4ed2-83e5-e672f7c857f9.png">

Reply:
Agreed. The issue we're seeing is that 2.8.3 does not pass the tests and 2.8.2 loads (unexpectedly given that it's yanked), but does pass the tests. The goal is NOT to release 2.8.2 for the reason you pointed out. I just got to this decision point and pushed it to the branch as a stopping point for discussion. We're discussing internally how to proceed.